dev_ops/sql/Tables/CreatePaymentInfo.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE payment_info (
+    id BIGSERIAL PRIMARY KEY,
+	user_id BIGINT REFERENCES users (id),
+	paypal_username TEXT NOT NULL UNIQUE
```

Comment:
whats a use case for capturing the paypal username? Im guessing most/some peoples paypal names arent the same as twitch usernames. When do we display one over the other?

Reply:
Most likely the twitch/mixer/youtube username will not be the same as paypal. Won't we need to store this information so we can process their payments? Let me know if I'm wrong, but I thought this might be necessary.

Reply:
I dont think its necessary

Reply:
Going to remove this table for now...need to understand the payment scenario better to model the paypal transactions.